powerline/bindings/tmux/__init__.py

```diff
@@ -76,6 +76,8 @@ def get_tmux_version(pl):
 	version_string = get_tmux_output(pl, '-V')
 	_, version_string = version_string.split(' ')
 	version_string = version_string.strip()
+	if version_string == 'master':
+		return TmuxVersionInfo(math.inf, 0, version_string)
```

Comment:
I use 3.4 by default. And powerline supports 2.6+ and 3.2+, so there are lots of supported versions which dou not have `math.inf`.